.github/workflows/tests.yml

```diff
@@ -63,14 +63,14 @@ jobs:
       fail-fast: false
       matrix:
         # test all lower versions (w/ stable conda) and upper version (w/ canary conda)
-        python-version: ['3.8', '3.9']
+        python-version: ['3.8', '3.9', '3.10']
```

Comment:
do we still need 3.8.. now..

Reply:
yes, Python 3.8 support hasn't been dropped yet so it still needs testing

Reply:
is the plan to drop it for next release round??

Reply:
at this time we only drop Python 3.8 once it reaches EOL/when support is dropped from the defaults channel

Reply:
makes sense